opal/templates/partials/_teams_list.html

```diff
@@ -1,12 +0,0 @@
-<h4>
-  <i class="fa fa-users"></i> Teams
```

Comment:
yep, except that's in v0.9.0. We could leave it in just so that we can add in a warning for it in 0.9.1 but I'm not sure its worth it